src/pages/candidate-details/candidate-details.ts

```diff
@@ -29,6 +29,11 @@ interface CandidateDetailsPageState {
   business: Business;
 }
 
+enum driverType {
+  R = 'Rider',
+  D = 'Driver',
+}
+
```

Comment:
Would be nice to make use of this enum in the spec file, would require moving out to it's own file and referenced by both , but it's very minor.

Reply:
enum moved out along with the function to its own helper file